packages/protocol/contracts/common/UsingFixidity.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.5.8;
+
+import "fixidity/contracts/FixidityLib.sol";
+
+
+contract UsingFixidity {
+  using FixidityLib for int256;
+
+  int256 public constant FIXED1 = 1000000000000000000000000;
+  int256 public constant FIXED_HALF = 500000000000000000000000;
```

Comment:
not a big fan of this name... what is the half a of a fixed? I've seen it used in Governance, maybe it's 50%?

Reply:
Any other suggestions? This name felt pretty natural to me. Can add a clarifying comment.

Reply:
Maybe, so this is actually the number: `0.5`